src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -650,8 +650,33 @@ public async Task<GitFile> GetGitTreeItem(string path, TreeItem treeItem, string
         /// <returns>Git file with tree item contents.</returns>
         private async Task<GitFile> GetGitItemImpl(string path, TreeItem treeItem, string owner, string repo)
         {
-            Octokit.Blob blob = await ExponentialRetry.Default.RetryAsync(
-                                async () => await Client.Git.Blob.Get(owner, repo, treeItem.Sha),
+            Blob blob = await ExponentialRetry.Default.RetryAsync(
```

Comment:
Could possibly put the logic for dealing waiting until the Retry After Seconds into `Func<Exception, bool> isRetryable`  (line 681 below) since the point of ExponentialRetry is to catch exceptions for you, but I don't feel strongly as this is probably more readable.

Reply:
That's a good option, but I like how readable this is. If someone else feels more strongly about that, I'm happy to change it.